lib/inquisition/generators/html/templates/index.html.haml

```diff
@@ -351,9 +379,39 @@
           var chart_element = $('#errorsCount');
           var chart_doughnut = new Chart( chart_element, chart_doughnut_settings);
         }
-      }
 
-      function show_more_issues(){
-        var item = $('#show_more_issues')
-        console.log(item)
+        if ($('#routesCount').length){
```

Comment:
separate js files for each page and maybe for each data item on page and then group it by page